Cinecon/ChoiceMenu.cs

```diff
@@ -118,6 +127,154 @@ private void ChoiceSetup(List<KeyValuePair<string, Action>> preselected = null)
+                            }
+                        }
+                        else
+                        {
+                            if (!choices.Contains(choice) && !room.Seats.FirstOrDefault(x => $"{ x.Row}{ (x.Number < 10 ? "0" : "")}{ x.Number}" == choice.Key).IsTaken)
+                                choices.Add(choice);
+                            else
+                                choices.Remove(choice);
+                            if (!room.Seats.FirstOrDefault(x => $"{ x.Row}{ (x.Number < 10 ? "0" : "")}{ x.Number}" == choice.Key).IsTaken)
+                            {
+                                WriteAllMenu(indexY, indexX, choices, room: room, takenSeat: room.Seats.FirstOrDefault(x => $"{ x.Row}{ (x.Number < 10 ? "0" : "")}{ x.Number}" == choice.Key).IsTaken);
+                                Console.Clear();
+                            }
+                            WriteAllMenu(indexY, indexX, choices, room: room, takenSeat: room.Seats.FirstOrDefault(x => $"{ x.Row}{ (x.Number < 10 ? "0" : "")}{ x.Number}" == choice.Key).IsTaken);
+                        }
+                        break;
+                    case ConsoleKey.Backspace:
+                        Console.Clear();
+                        return Tuple.Create("Terug", room.Seats);
+                }
+            }
+        }
+        
+        private void WriteAllMenu(int indexY, int indexX, List<KeyValuePair<string, Action>> selectedChoices = null, Room room = null, bool seatsAmount = true, bool takenSeat = false)
+        {
+            ConsoleHelper.WriteLogo(ConsoleColor.Red);
+            ConsoleHelper.WriteBreadcrumb();
+
+            if (!string.IsNullOrEmpty(_text))
+                ConsoleHelper.ColorWriteLine(_text, _textColor);
+
+            foreach (var row in _2dChoices)
+            {
+                foreach (var choice in row)
+                {
+                    bool seatIsTaken = false;
+                    bool choiceIsSelected = selectedChoices != null && selectedChoices.Contains(choice);
+                    if (new[] { "Terug", "Ga door" }.Any(x => x == choice.Key))
+                    {
+                        Console.WriteLine();
+                        ConsoleHelper.ColorWrite(choice.Key == _2dChoices[indexY].ElementAt(indexX).Key ? $" > {choice.Key}" : $"   {choice.Key}", choice.Key == _2dChoices[indexY].ElementAt(indexX).Key ? ConsoleColor.Red : ConsoleColor.White);
+                    }
+                    else
+                    {
+                        seatIsTaken = room.Seats.FirstOrDefault(x => $"{ x.Row}{ (x.Number < 10 ? "0" : "")}{ x.Number}" == choice.Key).IsTaken;
+
+                        if (choice.Key == _2dChoices[indexY].ElementAt(indexX).Key)
+                            ConsoleHelper.ColorWrite($"   [{choice.Key}]", choiceIsSelected && !seatIsTaken ? ConsoleColor.Yellow : ConsoleColor.Red);
```

Comment:
Wanneer de stoel is groen is en je selecteert het, dan is het een beetje verwarrend als het geel wordt, want de breadcrumbs en text hebben dezelfde kleur. Ik heb donker geel geprobeerd en dat zag er volgens mij een stuk beter uit.